client/dom/ColorScale.ts

```diff
@@ -210,6 +209,12 @@ export class ColorScale {
 		this.data = this.data.map(d => Number(d.toFixed(2)))
```

Comment:
If the scale is 0.001 to 0.002, this `.toFixed(2)` will not work. Also, the formatting should only be applied to labels for axis or ticks, it should not modify the original data. As a general, we should try to avoid modifying the original input data/argument unless necessary or guaranteed to be harmless.

Reply:
i agree this must be addressed @creilly8